src/models/image-map.ts

```diff
@@ -51,8 +51,13 @@ export interface IImageHandlerStoreResult {
   displayUrl?: string;
   success: boolean;
 }
+interface IImageMap {
+  curriculumBaseUrl?: string;
```

Comment:
I think you could get rid of this property because it doesn't seem to be used by any of the handlers.